api/v2/sdks/client-side-identity.md

```diff
@@ -22,55 +44,55 @@ Include the following SDK script on the pages where you want to use UID2 to mana
 
 The high-level client-side workflow for establishing UID2 identity using the SDK consists of the following steps:
 
-1. [Initialize the SDK](#initopts-object-void) and specify a [callback function](#callback-function) to be called upon a successful completion of the step.
-2. Wait for the SDK to invoke the callback function. The callback function indicates the identity availability:
-	- If the identity is available, the [background token auto-refresh](#background-token-auto-refresh) is set up.
-	- If not, the reason for its unavailability is specified.
-3. Based on the identity [state](#workflow-states-and-transitions), the SDK does the following:
-	- If a valid identity is available, the SDK ensures the identity is available in a [first-party cookie](#uid2-cookie-format).
-	- If the identity is unavailable, the SDK takes the appropriate action based on whether identity is refreshable or not. For details, see [Workflow States and Transitions](#workflow-states-and-transitions).
-4. Handle the identity based on its state:
+1. Publisher: Initialize the SDK using the [init](#initopts-object-void) function and specify a [callback function](#callback-function) to be called upon successful completion of the step.
+2. Publisher: Wait for the SDK to invoke the callback function. The callback function indicates the identity availability:
+	- If the identity is available, the SDK sets up a [background token auto-refresh](#background-token-auto-refresh).
+	- If the identity is unavailable, the reason for its unavailability is specified in the response.
```

Comment:
In this case, the reason is specified "on the object passed to the [callback function](callback-function)." or words to that effect.

Reply:
Update:
If the identity is unavailable, the reason for its unavailability is specified in the object passed to the callback function.